maximbircu/devtools-library
# Add Tool In-memory persistence

Closes: #33


### What has been done
1. Add an in-memory persistence mechanism, this way we always store all-new configuration value updates in memory so that the change is not missed as soon as the view is recycled;
2. Cover some more components with unit tests;
3. Fix the time dev tool click ripple.

### How to test
1. Open the app;
2. Select any source;
3. Update configuration value of the first tool;
4. Scroll the list till the end;
5. Scroll the list back to the tool you've updated;
6. Make sure the changes are preserved.
